src/System.IO.FileSystem/src/System.IO.FileSystem.builds

```diff
@@ -10,22 +10,13 @@
     </Project>
     <Project Include="System.IO.FileSystem.csproj">
```

Comment:
If we are not keeping the old builds, it does not make sense to keep the old stuff in `src/.csproj`.